OpenRA.Mods.Common/Widgets/ListLayout.cs

```diff
@@ -21,22 +21,24 @@ public class ListLayout : ILayout
 		public void AdjustChild(Widget w)
 		{
 			if (widget.Children.Count == 0)
-				widget.ContentHeight = widget.ItemSpacing;
+				widget.ContentHeight = 2 * widget.TopBottomSpacing - widget.ItemSpacing;
```

Comment:
But that wouldn't do the same thing. When adding the first item, both top and bottom spacings should be added.